package.json

```diff
@@ -18,7 +18,7 @@
   "main": "app/index.js",
   "repository": "h5bp/generator-server-configs",
   "scripts": {
-    "test": "mocha --timeout 50000"
+    "test": "mocha --timeout 60000"
```

Comment:
Yes sure, but it's good to know that a dependency you have doesn't work so you can file a bug and tell your user why it doesn't work ;)